{
	"cve": {
		"CVE_data_meta": {
			"ASSIGNER": "cve@kernel.org",
			"ID": "CVE-2024-50176"
		},
		"data_format": "MITRE",
		"data_type": "CVE",
		"data_version": "4.0",
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "In the Linux kernel, the following vulnerability has been resolved:\n\nremoteproc: k3-r5: Fix error handling when power-up failed\n\nBy simply bailing out, the driver was violating its rule and internal\nassumptions that either both or no rproc should be initialized. E.g.,\nthis could cause the first core to be available but not the second one,\nleading to crashes on its shutdown later on while trying to dereference\nthat second instance."
				}
			]
		},
		"problemtype": {
			"problemtype_data": [
				{}
			]
		},
		"references": {
			"reference_data": [
				{
					"name": "https://git.kernel.org/stable/c/87ab3af7447791d0c619610fd560bd804549e187",
					"url": "https://git.kernel.org/stable/c/87ab3af7447791d0c619610fd560bd804549e187"
				},
				{
					"name": "https://git.kernel.org/stable/c/fc71c23958931713b5e76f317b76be37189f2516",
					"url": "https://git.kernel.org/stable/c/fc71c23958931713b5e76f317b76be37189f2516"
				},
				{
					"name": "https://git.kernel.org/stable/c/afd102bde99d90ef41e043c846ea34b04433eb7b",
					"url": "https://git.kernel.org/stable/c/afd102bde99d90ef41e043c846ea34b04433eb7b"
				},
				{
					"name": "https://git.kernel.org/stable/c/7afb5e3aa989c479979faeb18768a67889a7a9c6",
					"url": "https://git.kernel.org/stable/c/7afb5e3aa989c479979faeb18768a67889a7a9c6"
				},
				{
					"name": "https://git.kernel.org/stable/c/9ab27eb5866ccbf57715cfdba4b03d57776092fb",
					"url": "https://git.kernel.org/stable/c/9ab27eb5866ccbf57715cfdba4b03d57776092fb"
				}
			]
		}
	},
	"impact": {},
	"configurations": {
		"CVE_data_version": "4.0"
	},
	"lastModifiedDate": "2024-11-08T19:01Z",
	"publishedDate": "2024-11-08T06:15Z"
}
